packages/sol-rayz/src/getParsedAccountByMint.ts

```diff
@@ -0,0 +1,27 @@
+// @ts-nocheck
+import { clusterApiUrl, Connection, Commitment, PublicKey } from '@solana/web3.js';
+import { TOKEN_PROGRAM } from './config/solana';
+
+export const createConnectionConfig = (clusterApi = clusterApiUrl('mainnet-beta'), commitment = 'confirmed') =>
+  new Connection(clusterApi, commitment as Commitment);
+
+export const getParsedAccountByMint = async ({ mintAddress, connection = createConnectionConfig() }) => {
+  try {
+    const res = await connection.getParsedProgramAccounts(new PublicKey(TOKEN_PROGRAM), {
+      filters: [
+        { dataSize: 165 },
```

Comment:
wondering what is `165` stands for?
maybe leave some comment about it

Reply:
<img width="775" alt="Screenshot 2021-11-16 at 15 11 37" src="https://user-images.githubusercontent.com/18005988/141991364-6bced992-d4e5-417f-b6d1-c9c1c9e9c9c0.png">
I will probably write an article about it. Problematically explain it in 2 words. In short, explanation will be 
The "dataSize": 165 filter selects all Token Accounts

Reply:
ah. I got what it is. I think something like
`length of Account Struct in bytes` might be enough as a comment